GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java

```diff
@@ -193,6 +194,19 @@ public GVRActivity getActivity() {
     }
 
     /**
+     * Get the {@link GVRImporter} to use for loading assets.
+     * 
+     * A {@code GVRImporter} loads models asynchronously from your applications's
+     * local storage or the network.
+     * 
+     * 
+     * @return The asset loader associated with this context.
+     */
+    public GVRImporter getAssetLoader() {
```

Comment:
Also, there seems to be two conflicting ways of getting an importer object - using this call getAssetLoader or creating one using GVRImporter importer = new GVRImporter(context). This might be a little too confusing. We need to decide on one or the other.